src/Simplify_Add.cpp

```diff
@@ -49,100 +49,99 @@ Expr Simplify::visit(const Add *op, ExprInfo *bounds) {
             return rewrite.result;
         }
 
-        if (EVAL_IN_LAMBDA
```

Comment:
Let's skip Simplify_*.cpp. It makes a mess.

Reply:
Sure, but we should insert no-clang-format here (and elsewhere) to avoid that; I'd like to get to a point where we can run clang-format as a presubmit check, so avoiding special cases is good.